File: 924-fair-candy-swap/fair-candy-swap.java
class Solution {
    public int[] fairCandySwap(int[] aliceSizes, int[] bobSizes) {
        int sumA = 0;
        int sumB = 0;
        for(int i: aliceSizes) sumA+=i;
        for(int j: bobSizes) sumB+=j;
        int diff = (sumA-sumB)/2;
        for(int k: aliceSizes){
            for(int l: bobSizes){
               if(k == l+diff) return new int[]{k,l};
            }
        }
        return new int[]{-1,-1};
    }
}

File: 924-fair-candy-swap/fair-candy-swap.py
class Solution:
    def fairCandySwap(self, aliceSizes: List[int], bobSizes: List[int]) -> List[int]:
        a = sum(aliceSizes)
        b = sum(bobSizes)
        diff = (b-a)//2
        c = set(bobSizes)
        for i in aliceSizes:
            d = i+diff
            if d in c:
                return [i,d]
        